tests/test_checks.py

```diff
@@ -17,18 +17,44 @@ def change_test_dir(request, monkeypatch):
     monkeypatch.chdir(request.fspath.dirname)
 
 
+def job_file_generator(
+    program: str = None,
+    header_extra: str = None,
+    jobname: str = "TEST",
+    foldername="TESTFOLDER",
+    nop="NOP\n",
+    end="END\n",
+):
+    header = "/JOB\n"
+    if jobname is not None:
+        header += f"//NAME {jobname}\n"
+    if foldername is not None:
+        header += f"///FOLDERNAME {foldername}\n"
+    if header_extra is not None:
+        header += header_extra
+
+    if program is None:
+        program = """'A comment
+COMMAND\n"""
+
```

Comment:
If program is not `None`, it doesn't add a default command line.  In most cases in the following tests it program is not `None`.